PowerPointLabs/PowerPointLabs/CaptionsLab/NotesToCaptions.cs

```diff
@@ -17,6 +17,18 @@ namespace PowerPointLabs.CaptionsLab
     internal static class NotesToCaptions
     {
 #pragma warning disable 0618
+        private static bool removeCaptionsEnabled = true;
```

Comment:
Since this is a C# project, maybe it would be better to use properties instead? (unless there is any inconsistency in other classes)

private bool isRemoveCaptionsEnabled;
public string IsRemoveCaptionsEnabled {
   get { return isRemoveCaptionsEnabled; }
   set { isRemoveCaptionsEnabled = value; }
}

Reply:
Good point, I will use properties instead.

Maybe it'll be good to add the agreed style on the https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-CSharp.adoc doc?

Reply:
Yes, create a new issue for that